src/niftyone/figures/multi_view.py

```diff
@@ -0,0 +1,49 @@
+"""Generators associated with multi-view."""
+
+from pathlib import Path
+
+import pandas as pd
+
+from niclips.figures import multi_view
+from niftyone.figures.generator import ViewGenerator, register
+
+
+@register("three_view")
+class ThreeViewGenerator(ViewGenerator):
+    def generate(self, record: pd.Series, out_dir: Path, overwrite: bool) -> None:
+        self.generate_common(
```

Comment:
The repeated use of `generate_common` feels like it could be simplified.

Reply:
Hmm, if there is a way to use `partial` with it, that could work. Will need to look into this.

Reply:
I can mess around with it a bit and work up an alternative in a separate branch for you to weigh in on.

Reply:
Okay, from the repeated use of `generate_common` and `generate` front, this should be fixed in 
6de7f62. 

I haven't touched the registry - I know there was some discussion around refactoring that as well. Do we want to try to add that in here or push that down the road a little bit to try to get some additional figures generated for derivative data? While I don't love the use of decorators here anymore (given the unexpected behaviour I had encountered trying to fix `generate`), I don't think it is doing anything breaking.